### PR TITLE
Add a runbook for the CaDeT integration

### PR DESCRIPTION
**Pull request by MatMoore**
This documents the metadata and tags we need to ingest from DBT into the catalgoue, and render useful contact information in our custom frontend.

### CODE REVIEW

File: source/documentation/runbooks/008-cadet-integration.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+The data catalogue uses DBT as a source of metadata about the Analytical Platform.
+
+By default, all models and sources will be ingested into the Datahub catalogue, but they will not be shown in the Find MOJ Data service.
+
+## Make a model or source visible
+
+To make a model or source visible in the Find MOJ Data frontend, set the `dc_display_in_catalogue` tag.
+
+For example, in `dbt_project.yaml` you can include
```
[the remark this answers is not in the document]
Comment (MatMoore) [reply source/documentation/runbooks/008-cadet-integration.html.md.erb:18]: Both are options, but I think `dbt_project.yaml` works best
- you can set the metadata at a directory level, so you don't have to repeat things more than necessary
- you just have to check one file to see what will and will not be included in the catalogue
- the data owner information is in one place - so if there are updates to governance you will just change one file

But meta and tags can be set in any of 3 places:

- Using a [config() Jinja macro](https://docs.getdbt.com/reference/dbt-jinja-functions/config) within a model, snapshot, or test SQL file
- Using a [config property](https://docs.getdbt.com/reference/resource-properties/config) in a .yml file
- From the [dbt_project.yml file](https://docs.getdbt.com/reference/dbt_project.yml), under the corresponding resource key (models:, snapshots:, tests:, etc)

https://docs.getdbt.com/reference/configs-and-properties#where-can-i-define-configs

File: source/documentation/runbooks/008-cadet-integration.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+title: Configure DBT (Create a Derived Table) to send metadata to the catalogue
+last_reviewed_on: 2024-06-04
+review_in: 3 months
+---
+
+# <%= current_page.data.title %>
+
+The data catalogue uses DBT as a source of metadata about the Analytical Platform.
+
+By default, all models and sources will be ingested into the Datahub catalogue, but they will not be shown in the Find MOJ Data service.
```
Comment (murdo-moj) [inline source/documentation/runbooks/008-cadet-integration.html.md.erb:12]: We'll need to amend the ingestion recipe to make this the case

Comment (MatMoore) [reply source/documentation/runbooks/008-cadet-integration.html.md.erb:12]: Got a PR for that here https://github.com/moj-analytical-services/create-a-derived-table/pull/1568

Comment (murdo-moj) [reply source/documentation/runbooks/008-cadet-integration.html.md.erb:12]: Our ingestion recipe also needs an amendment so we pull in all tables I mean.

Comment (MatMoore) [reply source/documentation/runbooks/008-cadet-integration.html.md.erb:12]: Ah right - I'll follow up on that

Comment (MatMoore) [reply source/documentation/runbooks/008-cadet-integration.html.md.erb:12]: Follow up ticket: https://github.com/ministryofjustice/data-catalogue/issues/145

Comment (LavMatt) [reply source/documentation/runbooks/008-cadet-integration.html.md.erb:12]: Just need to remove the allow regex block

Comment (MatMoore) [reply source/documentation/runbooks/008-cadet-integration.html.md.erb:12]: @LavMatt I've included that in the above ticket ☝🏻

**Pull request merged**